src/App.vue

```diff
@@ -39,12 +36,15 @@ export default {
       await initWallet();
       const hasLoggedIn = await isLoggedIn();
       if (hasLoggedIn) {
-        await fetchUserDetails();
+        store.dispatch("showFullScreenLoader", "Fetching account details...");
+        setTimeout(async () => {
```

Comment:
@shrinathprabhu Why do we need this `setTimeout`?

Reply:
For some unknown reason it becomes unresponsive for couple of milliseconds. Tried using nextTick. Most probably parent app and iframe use the same thread, not sure. Need to investigate more on this.